src/util/string.h

```diff
@@ -15,6 +16,15 @@
 #include <string>
 #include <vector>
 
+[[nodiscard]] inline std::vector<std::string> SplitString(const std::string& str, char sep)
+{
+    std::vector<Span<const char>> span_strs = spanparsing::Split(str, sep);
+    std::vector<std::string> result;
+    for (const auto& span_str : span_strs)
```

Comment:
nit: missing `{`. Also, could move the impl to the cpp file?

```suggestion
    for (const auto& span_str : span_strs) {
```

Reply:
Thanks for the review, done (was surprised to see that the string.cpp is actually empty on master).